miniml/compiler/compile.scm

```diff
@@ -1473,8 +1520,10 @@
 (define (compile-args env stacksize l) (compile-expr-list env stacksize (reverse l)))
 
 
-(define (vhash-map proc l) (alist->vhash (map (lambda (kv) (cons (car kv) (proc (car kv) (cdr kv)))) (vlist->list l))))
-(define (vhash-filter-map proc l) (alist->vhash (filter-map (lambda (kv) (let ((r (proc (car kv) (cdr kv)))) (if r (cons (car kv) r) r))) (vlist->list l))))
```

Comment:
Note: I removed `filter-map` because it was easier to implement at once at the `bindings` level and not used anywhere else, but I regret not, I may reinstate it.

Reply:
Done.